flocx_market/cmd/api.py

```diff
@@ -11,19 +11,35 @@
 #    under the License.
 import sys
 
-from oslo_service import service
+# from oslo_service import service
 
 from flocx_market.api import service as wsgi_service
 from flocx_market.common import service as flocx_market_service
 import flocx_market.conf
 
+from flocx_market.api import app
+from flask_migrate import Migrate
+from flocx_market.db.orm import orm
+
+
 CONF = flocx_market.conf.CONF
+flocx_market_service.prepare_service(sys.argv)
+
+
+application = app.create_app(app_name='flocx-market')
+migrate = Migrate(application, orm)
```

Comment:
these lines - 26-30 - should be *inside* the main function

Reply:
Hey @tzumainn, to run the integration and system tests, I need the application context (used in tests/base_test.py), if I put these 2 lines in the main function I don't have access to it from other files. Any suggestions on how to resolve that?

Reply:
Can you just write a function that returns it, instead of importing it?

Reply:
that will probably work for now, I'll do that